### DISCUSSION TITLE
More documentation on createTenant createProject mutation

### DISCUSSION THREAD

**Discussion by eduardBM** [hasura/graphql-engine#8294] [Question]
I'm trying to automate the createProject functionality, and i can't find detailed information on the mutation.
Started from this https://hasura.io/docs/latest/graphql/cloud/api-reference.html#create-a-project but i would like to customize the name, the DB connection params, etc.

Where can i find the detailed documentation for this mutation?

**Comment by aarong506** [comment]
any update on this?
